app/models/concerns/wp_cache.rb

```diff
@@ -48,11 +56,31 @@ def create_or_update(wp_type, wp_id, preview = false)
     # the `update_wp_cache` instance method.
     # Removes records with unknown IDs.
     #
-    def create_or_update_all
-      wp_json = get_from_wp_api self.to_s.underscore.pluralize
+    # TODO (dunyakirkali) clean up
+    def create_or_update_all_paginated
```

Comment:
Assignment Branch Condition size for create_or_update_all_paginated is too high. [21.26/15]